README.md

```diff
@@ -75,10 +85,12 @@ Create a `secretproviderclasses` resource to provide provider-specific parameter
 
     ```
 
-    | Name           | Required | Description                                                     | Default Value |
-    | -------------- | -------- | --------------------------------------------------------------- | ------------- |
-    | provider       | yes      | specify name of the provider                                    | ""            |
-    | usePodIdentity | no       | specify access mode: service principal or pod identity          | "false"       |
+    | Name                   | Required | Description                                                     | Default Value |
+    | --------------         | -------- | --------------------------------------------------------------- | ------------- |
+    | provider               | yes      | specify name of the provider                                    | ""            |
+    | usePodIdentity         | no       | specify access mode: service principal or pod identity          | "false"       |
+    | useVMManagedIdentity   | OPTIONAL available for version > 0.0.4       | specify access mode: use a service principal or pod identity or vm managed identity    |  "false"|
```

Comment:
nit: Required should be `no`. maybe consider moving `available for version > 0.0.4 ` to the Description?

Reply:
The description should say `specify access mode to enable use of VM's managed identity`

Reply:
> nit: Required should be `no`. maybe consider moving `available for version > 0.0.4 ` to the Description?

We may move it to the description like  (available for version > 0.0.4 specify)

P.S. we will need to change the rest of properties as well to be consistent